library/dash/src/main/java/com/google/android/exoplayer2/source/dash/DefaultDashChunkSource.java

```diff
@@ -818,6 +818,9 @@ public long getLastAvailableSegmentNum(long nowUnixTimeUs) {
     }
 
     public boolean isSegmentAvailableAtFullNetworkSpeed(long segmentNum, long nowPeriodTimeUs) {
+      if (segmentIndex.isExplicit()) {
+        return true;
```

Comment:
Could you add a comment here to explain why this is done? Because technically the code is still correct as it is and this new addition is just a workaround because we don't support detecting unavailable segments for explicit indices.

For example:
"// We don't support segment availability for explicit indices. Hence, also assume all segments in explicit indices are always available at full network speed even if they end in the future."

Reply:
Done :)